### ISSUE TITLE
Scale price chart properly

### ISSUE THREAD

**Issue by sea212** [zeitgeistpm/ui#273]
Currently the chart always shows values up to 1 ZTG on the y axis, even if the full range of the y axis is not utilized. I think it would make more sense to show the range between the maximum and minimum value, or at least allow the user to change the scale of the y axis. The more outcomes a market has, the more the chart could benefit from this feature. The market [Who will win the 2022 FIFA World Cup?](https://ztg-staging.vercel.app/markets/14) is a good example for that:

![image](https://user-images.githubusercontent.com/15378334/190392630-a2641858-fbf3-49b8-8396-a965b2345f96.png)

As we can see in the image above, much information gets visually lost due to the representation.
Applying functions on the values of the chart might also be helpful, such as showing the logarithmic values.

**Comment by Nathanpmyoung** [comment]
I suggest that it would make more sense if it was "Implied % odds". That feels more like how users will think.

**Comment by sea212** [comment]
I agree @Nathanpmyoung . In that case a normalization must happen, such that `sum(outcome_percentages) = 100`, since currently arbitrage is possible (i.e. `sum(outcome_percentages)` can be greater than 100).

**Comment by Nathanpmyoung** [comment]
Sure, seems reasonable. and it’s fine because it says, implied

Yours,

Nathan Young

*M* 07891 020271
*W* nathanpmyoung.com
*T  ***@***.*** <https://twitter.com/nathanpmyoung>

Personal newsletter <https://tinyletter.com/nathanpmyoung>

If I can put you in touch with someone, please ask.

Sent via Superhuman iOS ***@***.***>

**Comment by Robiquet** [comment]
We used to use 0 to max values though @lsaether requested it be adjusted to this, do you still feel the same way?

**Comment by Nathanpmyoung** [comment]
I think the manifold graph in the description is the best way. Stacks of colours for each outcome.
